src/resolve/resolved-dns-query.c

```diff
@@ -508,6 +508,55 @@ DnsQuery *dns_query_free(DnsQuery *q) {
         return mfree(q);
 }
 
+/* Almost all of this is not my code, it's help from @poettering */
+static int manager_validate_and_mangle_question(Set *types, DnsQuestion **question) {
```

Comment:
The main target argument is `DnsQuestion`, and this does not take `Manager` object.
Hence, please rename and reorder argument:
```c
static int validate_and_mangle_question(DnsQuestion **question, Set *types) {
```